bigtable/google/cloud/bigtable/row_set.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""User-friendly container for Google Cloud Bigtable RowSet """
+
+
+from google.cloud._helpers import _to_bytes
+
+
+class RowSet(object):
+    """ Convenience wrapper of google.bigtable.v2.RowSet
+
+        Useful for creating a set of row keys and row ranges, which can
+        be passed to yield_rows method of class:`.Table.yield_rows`.
+    """
+
+    def __init__(self):
+        self.row_keys = []
+        self.row_ranges = []
+
+    def add_row_key(self, row_key):
+        """Add row key to row_keys list.
+
+        :type row_key: bytes
+        :param row_key: The key of a row to read
+        """
+        self.row_keys.append(row_key)
+
+    def add_row_range(self, row_range):
```

Comment:
It would be great to have something like:

```
    def add_row_range(self, start_key=None, end_key=None,
                                     start_inclusive=True, end_inclusive=False):
````

Reply:
Added a new method : add_row_range_from_keys()